contracts/account-history-contract/src/action/query/get_liquid_assets.rs

```diff
@@ -49,3 +52,26 @@ fn get_info(list_info: &Vec<CoinValue>, denom: &String) -> (Decimal, Decimal) {
         None => (Decimal::zero(), Decimal::zero()),
     }
 }
+
+fn eden_to_liquid_asset(generator: AccountSnapshotGenerator, &deps: &Deps<ElysQuery>, user_address: String) -> StdResult<LiquidAsset> {
```

Comment:
Is it ok to add this function here or would it be better to make it part of the `AccountSnapshotGenerator`? If it's needed in the portfolio queries (not sure yet). It would need to be moved somewhere else.

Are these params and its references ok/

Reply:
Yes, this is needed in the portfolio queries as well. I have to define it somewhere else